index.html

```diff
@@ -0,0 +1,332 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="wrapper">
+        
+        <header class="header-2">
+            
+                <a href="./index.html" class="my-logo">My Logo</a>
+                <nav class="navbar">
+                    <ul>
+                        <li class="menu-bar">                            
+                            <button type="button" class="bar"><i class="fa fa-bars"></i></button>
+                        </li>
+                    </ul>
+                </nav>
+            
+        </header>
+        
+        <div class="container-homepage">     
+            <section id="#about" class="section-home">
+                <div class="welcome">
+                    <h1 class="name-welcome">
+                        I’m Amare <br> Glad to see <br> you!
+                    </h1>                
+                </div>
+                <div class="job-description">
+                    <p class="description-of-me">
+                        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                
+                <div class="lets-connect">
+                    <p class="lets-connect-statement">Let's connect</p>    
+    
+                    
+                    <ul class="social-networks">
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-twitter"></i>
+                            </a>
+                        </li>
+                        
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-linkedin"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-medium"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-github"></i>
+                            </a>
+                        </li>
+                       <li>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-angellist"></i>
+                        </a>
+                       </li>
+                    </ul>                    
+                </div>
+        
+            </section>
+        </div>
+        <!---------mobile-finish-version--------->
+        <section id="works" class="section-works">
+            <!-----card-works for Tonic------->
```

Comment:
- [ ] Please reduce the spacing before the works section to match the Figma design. Also, please add a background color to the body or wrapper of the web page to match the design template

*Your design*
![2](https://user-images.githubusercontent.com/94551484/186516512-2ca74746-c320-42ae-becc-128992dd2c5c.png)

*Figma template*
![bbb](https://user-images.githubusercontent.com/94551484/186516531-8cca4c46-efb2-485d-9700-4d57f1611600.png)
.